### PR TITLE
Test types

### PR DESCRIPTION
**Pull request by mkp13**
Create `testTypes` module for type testing

### CODE REVIEW

File: lib/countConsonants.js [hunk 1]
```diff
@@ -6,7 +6,16 @@
...
+  if (typeof str !== "string") throw new TypeError("The value is not a string.");
+
+  let consonantsCount = 0;
+  var i = str.length;
+  while (i--) {
+    if(!(str[i].match(/[aeiou]/)))
+      consonantsCount++;
+  }
+
+  return consonantsCount;
```
Comment (ktilcu) [inline lib/countConsonants.js]: This method has been claimed by someone else. Sorry.

File: lib/countVovels.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Write a function that accepts a string and returns the number of vovels in the string
+ * @param {String} str
+ * @return {Number}
+ */
+
+function vovelCount(str) {
```
Comment (ktilcu) [inline lib/countVovels.js]: Should this be vowel?

File: test/countConsonants.test.js [hunk 3]
```diff
@@ -15,9 +16,8 @@ describe("countConsonants", () => {
     expect(countConsonants("")).toEqual(0);
   });
   test("invalid parameters", () => {
-    expect(() => countConsonants({obj:"val"})).toThrowError(TypeError);
-    expect(() => countConsonants(["array"])).toThrowError(TypeError);
-    expect(() => countConsonants(undefined)).toThrowError(TypeError);
-    expect(() => countConsonants(null)).toThrowError(TypeError);
+    testTypes
+      .filter(t => typeof t != "string")
+      .forEach(t => expect(() => countConsonants(t)).toThrowError(TypeError));
```
Comment (ktilcu) [inline test/countConsonants.test.js]: I think I would rather the tests be explicit. The types array could be updated separately and change how this test works.

**Comment by ktilcu** [comment]
I'm going to close this PR. As soon as you find another function to fix we can get that merged! Thanks so much for working on this and I hope to get your code in soon!

File: lib/countVovels.js (hunk 2 above)
Comment (mkp13) [reply lib/countVovels.js]: Yes indeed!

**Comment by mkp13** [comment]
Thanks so much for response!

**Pull request closed without merging**